packages/manager/apps/telecom/src/app/telecom/pack/move/eligibility/lineNumber/move-eligibility-lineNumber.controller.js

```diff
@@ -24,21 +24,39 @@ export default class {
     this.phoneNumberRegex = '^0[1-5]([\\s\\-]?([0-9]){2}){4}$';
   }
 
+  /**
+   * Test line copper for the line number and status
+   * @param { String } lineNumber phone number to check eligibility
+   * @param { String } status status of the line: 'active' | 'inactive'
+   */
+  testLine(lineNumber, status) {
+    if (this.isReseller) {
+      // If current offer is a reseller offer,
+      // launch eligibility test line for partners (reseller)
+      return this.OvhApiConnectivityEligibility.v6().testLinePartners(
+        this.$scope,
+        {
+          lineNumber,
+          status,
+        },
+      );
+    }
+    return this.OvhApiConnectivityEligibility.v6().testLine(this.$scope, {
+      lineNumber,
+      status,
+    });
+  }
+
   /**
    * Retrieve line copper for the line number and status
    * @param { String } lineNumber phone number to check eligibility
    * @param { String } status status of the line: 'active' | 'inactive'
    */
   getLineCopper(lineNumber, status) {
-    return this.OvhApiConnectivityEligibility.v6()
-      .testLine(this.$scope, {
-        lineNumber,
-        status,
-      })
-      .catch((error) => {
-        this.loading = false;
-        this.TucToast.error(error);
-      });
+    return this.testLine(lineNumber, status).catch((error) => {
```

Comment:
No `then` ?

Reply:
the `then` is resolved by the calling method.